tests/data_reviews/test_data_reviews_mutations.py

```diff
@@ -263,8 +265,13 @@ def test_update_data_review(
         assert resp.json()["errors"][0]["message"] == "Not allowed"
         return
 
-    # Check expected end_state
     data_review.refresh_from_db()
+
+    # Check that validation results were cleared
+    with pytest.raises(ValidationResultset.DoesNotExist):
+        assert data_review.validation_resultset
```

Comment:
not entirely sure what to suggest, but I think this does it?

```suggestion
    with pytest.raises(ValidationResultset.DoesNotExist):
        _ = data_review.validation_resultset
```